test/controllers/drivers_controller_test.rb

```diff
@@ -6,99 +6,186 @@
   describe "index" do
     it "responds with success when there are many drivers saved" do
       # Arrange
+      Driver.create(
+        {
+          name: "June Bug",
+          vin: "UTGFEE6457426GYR",
+          available: true
+       }
+      )
       # Ensure that there is at least one Driver saved
-
+      
       # Act
-
+      get "/drivers"
       # Assert
+      must_respond_with :success
 
     end
 
     it "responds with success when there are no drivers saved" do
       # Arrange
       # Ensure that there are zero drivers saved
-
+      Driver.all.each do |driver|
+        driver.destroy
+      end
       # Act
-
+      get "/drivers"
       # Assert
-
+      must_respond_with :success
```

Comment:
Might be good to also verify that `Driver.all.count` is zero, just to internally validate your tests. :)